### PR TITLE
Submission structure showcase

### LINKED ISSUE #458: Structure Check visible when uploading submissions
So, when a student wants to submit a submission, they can see what their submission should look like.

### PR DESCRIPTION
**Pull request by bsilkyn**
Add the submission structure to StudentProjectView.vue and Submissions.vue

I also fixed the success message on error when submitting a submission.

closes #458

### CODE REVIEW

File: frontend/src/components/projects/ProjectStructure.vue [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+<template>
+    <div>
+        <label class="pb-2">
+            {{ t('views.projects.structureChecks.title') }}
+        </label>
+        <Tree class="w-100" selection-mode="single" :value="Project.getNodes(structureChecks)">
+            <template #default="{ node }">
+                <template v-if="node.obligated">
+                    <div class="w-full" v-tooltip="t('views.projects.structureChecks.obligatedExtensions')">
```
Comment (BramMeir) [inline frontend/src/components/projects/ProjectStructure.vue:25]: No such translation exists

File: frontend/src/components/projects/ProjectStructure.vue [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+                        <Chip
+                            class="m-1"
+                            :label="extension"
+                            :key="extension"
+                            v-for="extension in node.data.getObligatedExtensionList()"
+                        />
+                    </div>
+                </template>
+                <template v-else-if="node.blocked">
+                    <div class="w-full" v-tooltip="t('views.projects.structureChecks.blockedExtensions')">
```
Comment (BramMeir) [inline frontend/src/components/projects/ProjectStructure.vue:35]: No such translation exists

**Review by BramMeir — CHANGES_REQUESTED** [review]
Looks good

**Review by francisvaut — APPROVED** [review]
:apple: :green_apple:

**Review by BramMeir — APPROVED** [review]

**Pull request merged**